src/SourceCode.Clay/Number.cs

```diff
@@ -875,20 +954,30 @@ decimal IConvertible.ToDecimal(IFormatProvider provider)
             }
         }
 
-        double IConvertible.ToDouble(IFormatProvider provider)
+        public double ToDouble()
```

Comment:
New method override, optimized

Reply:
Copy the body of ToDouble and just do a straight cast.